src/util/logging.h

```diff
@@ -14,6 +14,14 @@ enum class LogLevel {
     Trace = 4, // for profiling etc.
 };
 
+enum class LogFlag {
+    None = 0,
```

Comment:
I suggest that we start using binary integer literals for flags:

```
None = 0b00,
LogToFile = 0b01,
DebugAssertBreak = 0b10,
```

Reply:
The issue with this notation is that we either have to reformat lines when we add new values or pad values with 31 zeroes.

How about using bitshift notation? 

```
None = 0
LogToFile = 1
DrbugAssertBreak = 1 << 1
...
```

Reply:
Even better

Reply:
Fixed